2024-frontend/src/views/WasteMeasurementTunnel/WasteMeasurementSteps/MeasurementPeriod.vue

```diff
@@ -51,10 +51,11 @@ const payload = reactive({})
 
 const afterStartDateValidator = (date) => {
   date = new Date(date)
-  return date - startDateAsDate.value >= 0
+  console.log(date, startDateAsDate.value)
+  return date >= startDateAsDate.value
 }
 
-const afterStartDate = helpers.withMessage("Faut être après la date de début", afterStartDateValidator)
+const afterStartDate = helpers.withMessage("La date de fin doit être après la date de début", afterStartDateValidator)
```

Comment:
```suggestion
const afterStartDate = helpers.withMessage("La date de fin ne peut pas être avant la date de début", afterStartDateValidator)
```
Est-ce qu'on ne tournerai pas la phrase au négatif ? Et on inclue le cas où la date de fin est égale à la date de début ?

Reply:
en fait la date de fin peut être égale à la date de début maintenant. pour permettre les évaluations d'1 seul jour.

Reply:
ok pour tourner au négatif. mais je vais modifier les erreurs renvoyées par le backend aussi dans ce cas là !

Reply:
ah pardon mon commentaire était pas clair : en disant "La date de fin doit être après la date de début", ça n'incluait pas le cas où elle était égale, alors que la version négative si. Mais si ça change trop côté back, peut-être pas besoin du négatif, mais un truc comme "La date de fin doit être égale ou supérieure à la date de début" ?